webui/src/pages/repositories/repository/settings/general.jsx

```diff
@@ -93,9 +93,17 @@ const SettingsContainer = () => {
                         <Form.Control readOnly value={repo.default_branch} type="text"/>
                     </Col>
                 </Row>
+                <Row>
+                    <Form.Label column md={{span:3}} className="mb-3">
+                        Read-only
+                    </Form.Label>
+                    <Col md={{span:4}} style={{paddingTop: 8 }}>
+                        <Form.Check checked={repo?.read_only} disabled={true} type="checkbox"/>
+                    </Col>
+                </Row>
```

Comment:
This confuses me visually, in 2 ways:
1. This is a checkbox that is always disabled.  When I see it I expect to be able to enable it, say if I were an admin.  But actually I can never change it.
1. In other places this indication appears as a badge.  It is less convenient for me as a user to have to read the same indication in two different ways, and then figure out that they both mean exactly the same.  I would prefer the same badge here, too!  (I actually did this when I saw the screenshot: I went to the code to figure out whether each "readonly" indication means something slightly different.)

Reply:
@arielshaqed, to be fair, the other fields are inputs, and they aren't editable either...  
But how about this?
![Screenshot 2024-01-01 at 16 02 50](https://github.com/treeverse/lakeFS/assets/110764839/3e0042b6-4397-4268-aa31-32e38894d09f)